.github/workflows/master-e2e.yaml

```diff
@@ -204,6 +204,7 @@ jobs:
     needs: [create-runner, pre-qase]
     runs-on: ${{ needs.create-runner.outputs.uuid }}
     env:
+      DS_CLUSTER_COUNT: 3
```

Comment:
Out of 3 if any one of the Downstream cluster may not start in time then it will impact on test execution.
Considering this risk, I would like to propose: 
* We can keep it to 1 or 2 for now.

This will not block merge of this PR but good to implement.

Reply:
If we keep it to 1 then I guess is the same place we are now.
2 would be the minimum, but given we have tests that will require 3 ds clusters and that there is a check for ds status like [here](https://github.com/rancher/fleet-e2e/pull/122/files#diff-e43af4652cde7ee6c6598d21986295cbdb9da436f79431923597c6fc2264fb13R375), I'd leave it as it is and check over time how reliable it behaves on ci.

Reply:
The cy tests won't run if one of the k3d doesn't run. I guess there is no big difference whether we provision 2 or 3. This can be fine tuned later IMO when we have an idea how it behaves long term.